pkg/tool/az-analyze/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
You can omit this file since the repo is already covered under the Apache 2.0 license.